docker-build.sh

```diff
@@ -0,0 +1,18 @@
+# Automatically retrieve the CUDA and torch version from the reference file env.yaml
+CUDA_VERSION=$(cat env.yaml | grep pytorch-cuda= | sed 's/.*pytorch-cuda=\([0-9]*\.[0-9]\).*/\1/')
+TORCH_VERSION=$(cat env.yaml | grep torch== | sed 's/.*torch==\([0-9]\.[0-9]\.[0-9]\).*/\1/')
```

Comment:
Could be interesting to be able to inject versions using env var like **PY4CAST_CUDA_VERSION** and **PY4CAST_TORCH_VERSION**. if not present we use the default values from the env.yaml. For ewc I need CUDA 11.8 I'd rather export an env var in my .bashrc than editing the script everytime.

Reply:
Yeh, good idea